hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTestSupport.java

```diff
@@ -92,14 +92,13 @@ protected void testNearCacheEviction(IMap<Integer, Integer> map, int size) {
 
     protected void assertNearCacheExpiration(final IMap<Integer, Integer> map, final int size, int expireSeconds) {
         final NearCacheStats statsBeforeExpiration = getNearCacheStatsCopy(map);
-        assertTrueEventually(new AssertTask() {
-            @Override
-            public void run() throws Exception {
-                assertTrue(format("we expected to have all map entries in the Near Cache or already expired (%s)",
-                        statsBeforeExpiration),
-                        statsBeforeExpiration.getOwnedEntryCount() + statsBeforeExpiration.getExpirations() >= size);
+        assertTrueEventually(() -> {
+            final NearCacheStats stats = getNearCacheStatsCopy(map);
```

Comment:
what about just using `getNearCacheStats` instead of `getNearCacheStatsCopy`? Seems copying stats is not required.